cli_tools/diagnostics/main_windows.go

```diff
@@ -26,6 +25,12 @@ import (
 	"time"
 )
 
+const (
+	eventLogsRoot = `C:\Windows\System32\winevt\Logs`
+	k8sLogsRoot   = `C:\etc\kubernetes\logs`
+	crashDump     = `C:\Windows\MEMORY.dmp`
```

Comment:
NIT: user can change the dump path: https://support.microsoft.com/en-us/help/254649/overview-of-memory-dump-file-options-for-windows

it will be safe to read the path from Registry .  But I don't think most people will do that.

Reply:
Added to TODO